.github/workflows/docker-autobuild.yml

```diff
@@ -0,0 +1,43 @@
+name: Docker Image Build
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
We aren't using the main branch for this repo, it was made years ago when the default was still "master".

We could rename the branch :woman_shrugging: